src/locales/en-US.json

```
@@ -39,40 +73,111 @@
+    "publicSwap": "Public swap",
+    "removeLiquidity": "Remove liquidity",
     "removing": "Removing",
     "rights": "Rights",
+    "search": "Search",
     "searchTypes": "Search name, symbol or address",
-    "similarPools": "There are similar pools available",
+    "selectToken": "Select token",
+    "selectTokens": "Select up to 8 tokens you’d like to have in the pool.",
+    "settings": "Settings",
+    "setSwapFee": "Set swap fee",
+    "setup": "Setup",
+    "setupProxy": "Setup proxy",
+    "setWeights": "Set weights",
+    "sharedPools": "Shared pools",
+    "shares": "Shares",
+    "similarPools": "There are similar pools available:",
+    "singleAsset": "Single asset",
     "slippage": "slippage",
+    "smartPoolController": "Smart pool controller",
+    "smartPools": "Smart pools",
+    "startBlock": "Start block",
+    "swapFee": "Swap fee",
+    "swapFeeLimits": "Swap fee range: 0.0001% to 10%",
     "swapFeePct": "Swap fee (%)",
+    "swaps": "Swaps",
     "toContinue": "to continue",
     "tokenSymbol": "Token symbol",
     "tokenName": "Token name",
+    "token": "Token",
     "tokens": "Tokens",
     "tokenWarning1": "• Do not add <b>deflationary tokens</b> or tokens with transfer fees.",
     "tokenWarning2": "• Do not add tokens whose transfer functions do not return <b>bool</b>.",
     "tokenWarning3": "• Any other <b>non-standard behavior from ERC20</b> may cause issues. Do your own research!",
     "tokenWarning4": "• Do not add tokens you don't trust, or whose rules for minting and blacklisting you don’t agree with.",
     "tokenWarning5": "• By checking this box, you agree that Balancer Labs is not liable for any losses you might incur as a direct or indirect result of adding liquidity to this pool.",
+    "toggle": "Toggle",
+    "totalSwapFee": "Total swap fee",
+    "totalSwapVolume": "Total swap volume",
     "totalValue": "Total value",
+    "tradeIn": "Trade in",
+    "tradeOut": "Trade out",
+    "transaction": "Transaction",
     "unlock": "unlock",
     "untrustedTokens": "This pool contains an untrusted token that may cause loss of funds. Do not deposit.",
+    "unwrap": "Unwrap",
+    "update": "Update",
+    "updateGradually": "Update gradually",
+    "value": "Value",
+    "version": "Version",
+    "volume": "Volume",
+    "volume24": "Volume (24h)",
+    "vulnerablePre": "If these",
+    "vulnerableMid": "s are not correct, you are vulnerable to loss of funds through arbitrage. You can click on",
+    "vulnerablePost": "to toggle manual input",
+    "waitingConfirmation": "Waiting for confirmation",
     "walletBalance": "Wallet balance",
     "weight": "Weight",
-    "weightConstraints": "Weights should be numbers from 2 to 98. Total weight should not exceed 100.",
+    "weightConstraints": "Denormalized weights should be numbers from 2 to 98. Total weight should not exceed 100.",
     "weights": "Weights",
-    "weightsPct": "Weights Percent"
+    "weightsPct": "Weights Percent",
```

Will be better to use Sentence case here instead of Title Case
```suggestion
    "weightsPct": "Weights percent",
```

---

Same here https://github.com/balancer-labs/pool-management-vue/pull/160/files#diff-65fd212bf11fb0785dba0e4951aca7edR13